fastapi_etag/dependency.py

```diff
@@ -12,6 +12,15 @@ class CacheHit(HTTPException):
     pass
 
 
+class PreconditionFailed(HTTPException):
+    pass
+
+
+class HeaderType(Enum):
+    IF_MATCH = "if-match"
+    IF_NON_MATCH = "if-non-match"
```

Comment:
Fix the typo here please: `IF_NONE_MATCH = "if-none-match"` :slightly_smiling_face:

Reply:
Fixed the typo and code formatting.